clients/src/main/java/org/apache/kafka/common/protocol/Errors.java

```diff
@@ -343,7 +344,8 @@
     INVALID_UPDATE_VERSION(95, "The given update version was invalid.", InvalidUpdateVersionException::new),
     FEATURE_UPDATE_FAILED(96, "Unable to update finalized features due to an unexpected server error.", FeatureUpdateFailedException::new),
     PRINCIPAL_DESERIALIZATION_FAILURE(97, "Request principal deserialization failed during forwarding. " +
-         "This indicates an internal error on the broker cluster security setup.", PrincipalDeserializationException::new);
+         "This indicates an internal error on the broker cluster security setup.", PrincipalDeserializationException::new),
+    UNKNOWN_TOPIC_ID(98, "The topic ID dose not exist", UnknownTopicIdException::new);
```

Comment:
nit: typo "dose" -> "does" I also had a slightly different message in my PR that implements this error. Either one is fine. https://github.com/apache/kafka/pull/9684/files

Reply:
Thank you, I think your message may be more appropriate.